cmd/plume/prune.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2016 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+
+	"github.com/coreos/mantle/platform/api/aws"
+	"github.com/coreos/mantle/platform/api/azure"
+)
+
+var (
+	days        int
+	pruneDryRun bool
+	cmdPrune    = &cobra.Command{
+		Use:   "prune [options]",
+		Short: "Prune old release images.",
+		Run:   runPrune,
+		Long:  `Prune old release images.`,
+	}
+)
+
+func init() {
+	cmdPrune.Flags().IntVar(&days, "days", 30, "Minimum age in days for files to get deleted")
+	cmdPrune.Flags().StringVar(&awsCredentialsFile, "aws-credentials", "", "AWS credentials file")
+	cmdPrune.Flags().StringVar(&azureProfile, "azure-profile", "", "Azure Profile json file")
+	cmdPrune.Flags().StringVar(&azureAuth, "azure-auth", "", "Azure Credentials json file")
+	cmdPrune.Flags().StringVar(&azureTestContainer, "azure-test-container", "", "Use test container instead of default")
```

Comment:
Maybe _use other container than the default_ would make it more clear?

Reply:
Done